airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.module.scss

```diff
@@ -0,0 +1,23 @@
+@use "../../../../../scss/colors";
+@forward "../../../../../views/layout/SideBar/SideBar.module.scss";
+
+.workspaceButton {
+  overflow: hidden;
+  cursor: pointer;
+  display: block;
+  height: 21px;
+  width: 100%;
+  border: 0;
+  border-radius: 10px;
+  background-color: rgba(255, 255, 255, 20%);
+  margin-top: 13px;
+  padding: 0 8px;
+  line-height: 21px;
+  font-size: 9px;
+  font-weight: 400;
```

Comment:
We can use `<Text />` component as a child of `<button>,` to control all the text styles

Reply:
Fixed! Thanks 😉

Reply:
Let's remove the custom text styles from this file since the font weight and size should be controlled by the `<Text/>` component now.